config/locales/ypk.yml

```diff
@@ -0,0 +1,135 @@
+﻿ypk:
+
```

Comment:
Please make this ```
ypk:

# Search.gov localizations for: Yup'ik (ypk)

# Default etc
```
see example at https://github.com/GSA/search-gov/blob/main/config/locales/ko.yml#L3